benchmarks/web-serving/db_server/entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Please double-check all the lines referencing mysql if it works after you change the name of db to mariadb.